curryreader.py

```diff
@@ -426,7 +426,7 @@ def read(inputfilename='', plotdata = 1, verbosity = 2):
         tixstop = contents.find('REMARK_LIST END_LIST')
         
         if tixstart != -1 and tixstop != 1 :
-            annotations = contents[tixstart:tixstop - 1].splitlines()
+            annotations = contents[tixstart+1:tixstop].splitlines()  # BUG (does not read correct lines) FIXED

```

Comment:
[nitpick] I'm unsure if the comment will still be helpful in 6 months. On the assumption that we only make changes to the reader if there was a demonstrable bug (i.e., no new features), the diff will tell us where the bugfixes were, and "does not read correct lines" seems (to me at least) inferrable from the change of indices in the slice. Feel free to disregard though, if you disagree.

```suggestion
            annotations = contents[tixstart+1:tixstop].splitlines()
```